packages/fiori/src/Login.js

```diff
@@ -0,0 +1,131 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+// import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+
+import Input from "@ui5/webcomponents/dist/Input.js";
+import Button from "@ui5/webcomponents/dist/Button.js";
+import Label from "@ui5/webcomponents/dist/Label.js";
+// Template
+
+import LoginTemplate from "./generated/templates/LoginTemplate.lit.js";
+
+// Styles
+import LoginCSS from "./generated/themes/Login.css.js";
+
+const metadata = {
+	tag: "ui5-login",
+	properties: {
+		email: {
```

Comment:
In productive use-case, we add docs for every public API, but for the test it's fine.